cmake/PACE_Docs.cmake

```diff
@@ -88,7 +88,7 @@ if (NOT sphinx-build-failed)
 
   else()
     add_custom_command(TARGET docs POST_BUILD
-      COMMAND sed -i -r "/\[NULL\]/d" "${Horace_DOCS_OUTPUT_DIR}/*html"
+      COMMAND sed -i "/][NULL]/d" "${Horace_DOCS_OUTPUT_DIR}/*html"
```

Comment:
```suggestion
      COMMAND sed -i -r "/\[NULL\]/d" "${Horace_DOCS_OUTPUT_DIR}/*html"
```
Ah, just noticed I forgot the `-r`, that explains why it's behaving weirdly in the docs build.

Reply:
Glad this is no longer needed.

Reply:
Yeah, this is some `sed` weirdness... so `\[` doesn't do what you expect and it was removing any line with `N`, `U`, or `L` in them... you need to use `][` to escape `[` and you don't need to escape `]`... 

https://stackoverflow.com/questions/27973519/

Reply:
Yeah, that's not quite the issue here. Maybe it's version differences? My experiments seem to show it failing. Who knows? Not important anyway. We can remove these whole chunks from the code.